dlup/tiling.py

```diff
@@ -1,4 +1,3 @@
-# coding=utf-8
```

Comment:
Is there a particular reason for removing the codings in many files?

Reply:
Yeah, it's not needed in python 3.